packages/code-studio/src/styleguide/ListViews.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useCallback, useState } from 'react';
+import { Grid, Item, ListView, ItemKey, Text } from '@deephaven/components';
+import { vsAccount, vsPerson } from '@deephaven/icons';
+import { Icon } from '@adobe/react-spectrum';
```

Comment:
I just picked one of the 2 supported. Is `illustration` preferable over `icon` ?

Reply:
Not 100% sure, but I think illustration must be added on the icons coming from `@spectrum-icons/illustrations/xxxxx`
https://react-spectrum.adobe.com/react-spectrum/ListView.html#complex-items

I think you are right that the default slot for `Icon` is `'icon'`. Unfortunately the ListView has to have `image` or `illustration` vs Picker which supports `icon`. Maybe a Spectrum bug but not sure.

Reply:
I think the expectation is you wouldn't put decorative icons in that slot, as it would be confusing with icons in the action slot (which I think in this example overflow because there is more than one)

![image](https://github.com/deephaven/web-client-ui/assets/1576283/53eb98f9-6df4-4267-a8d3-cdfd89127e91)

Where as the primary slot is for images/illustrations, not icons.

Reply:
I guess the lack of exporting Illustration might be an oversight:
https://github.com/adobe/react-spectrum/blob/main/packages/%40adobe/react-spectrum/src/index.ts#L32

Maybe should include `export {Icon} from '@react-spectrum/icon';`

Reply:
Perhaps, they also don't export `UIIcon`. Unsure if intentional or not. Could open a PR and see how they respond.